src/test/java/in/ashwanthkumar/gocd/artifacts/JanitorTest.java

```diff
@@ -25,55 +26,71 @@
 import static org.mockito.Mockito.when;
 
 public class JanitorTest {
-    Janitor janitor = new Janitor(mock(Action.class));
+    Action action = mock(Action.class);
+    GoCD client;
+    JanitorConfiguration config = new JanitorConfiguration();
+
+    @Before
+    public void before() {
+        client = mock(GoCD.class);
+        config = new JanitorConfiguration();
+    }
+
+    private Janitor janitor() {
```

Comment:
The janitor method?
To make the connection between the janitor, client and config more explicit?
I wouldn't, but I guess it's a matter of style.

I'm pretty new to open source and GitHub. If we need to make fixes befor you can merge, I guess they will have to be performed in the fork and a new pull request?